mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -37,7 +37,7 @@ class DistanceSensorItem {
 		send_tf(false),
 		sensor_id(0),
 		field_of_view(0),
-		orientation(-1),
+		orientation("LOCAL_NED"),
```

Comment:
But if I leave it as intenger, how can https://github.com/mavlink/mavros/pull/342/files#diff-57d2f68a3e0e5cfa2db14ac20d8845ddR117 be a string???